addons/website_livechat/static/tests/tours/website_livechat_no_new_session_with_hide_rule.js

```diff
@@ -0,0 +1,65 @@
+import { registry } from "@web/core/registry";
+import { contains, click } from "@web/../tests/utils";
+
+const sendFirstMessageSteps = [
+    {
+        trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+        run: "edit Hello, I need help!",
+    },
+    {
+        trigger: ".o-livechat-root:shadow .o-mail-Composer-input",
+        run: "press Enter",
+    },
+    {
+        trigger: ".o-livechat-root:shadow .o-mail-Message:contains('Hello, I need help!')",
+    },
+];
+registry.category("web_tour.tours").add("website_livechat_no_session_with_hide_rule", {
+    steps: () => [
+        {
+            trigger: ".o-livechat-root:shadow .o-livechat-LivechatButton",
+            run: "click",
+        },
+        ...sendFirstMessageSteps,
+        {
+            trigger: ".o-livechat-root:shadow [title='Close Chat Window (ESC)']",
+            run: "click",
+        },
+        {
+            trigger: ".o-livechat-root:shadow .o-livechat-CloseConfirmation-leave",
+            run: "click",
+        },
```

Comment:
Nitpick: `.o-livechat-root:shadow button:contains('Yes, leave conversation')` is a better selector: matches exactly what the user sees.